### DISCUSSION TITLE
会不会支持https访问呢

### DISCUSSION THREAD

**Discussion by rianjskis** [jeessy2/ddns-go#584] [Ideas]
在openwrt里使用如果给web页面开启了https访问，进入ddns-go就不显示界面报错😄😄😄

**Comment by jeessy2** [comment]
没看明白

**Comment by rianjskis** [comment]
> 没看明白

我给openwrt安装了证书自动跳转https访问后，在进去openwrt的ddns-go界面就会这样![IMG_20230303_134016_156.jpg](https://user-images.githubusercontent.com/86937445/222640584-e51a9e49-60e8-4261-adbc-5ed1911b98ad.jpg)

**Reply by jeessy2** [reply]
通过nginx转发是可以的。通过openwrt安装证书，没操作过，你能研究下原因不

**Reply by WaterLemons2k** [reply]
OpenWrt 默认只监听 443 端口，可以用 HTTP 访问 ddns-go 吗？

**Reply by rianjskis** [reply]
> OpenWrt 默认只监听 443 端口，可以用 HTTP 访问 ddns-go 吗？

可以的，我如果不强制http跳转https，在http下进入openwrt是可以访问ddns-go的，

**Reply by rianjskis** [reply]
> 通过nginx转发是可以的。通过openwrt安装证书，没操作过，你能研究下原因不

😄没试过nginx转发，我是自己在openwrt安装了acme申请的证书

**Reply by rianjskis** [reply]
> 通过nginx转发是可以的。通过openwrt安装证书，没操作过，你能研究下原因不

问过我这个固件作者，他说是ddns-go还目前没有HTTPS设置才会这样😄

**Reply by jeessy2** [reply]
自身是不带https的。带了https就复杂了

**Reply by rianjskis** [reply]
> 自身是不带https的。带了https就复杂了

好的，感谢解答😄😄

**Reply by jeessy2** [reply]
一般都是通过nginx转发，很少在应用程序中实现https。感兴趣可以看下nginx怎么在op系统中使用

**Reply by WaterLemons2k** [reply]
也可以使用 [Nginx Proxy Manager](https://nginxproxymanager.com)。

**Reply by rianjskis** [reply]
> 一般都是通过nginx转发，很少在应用程序中实现https。感兴趣可以看下nginx怎么在op系统中使用

好的

**Reply by rianjskis** [reply]
> 也可以使用 [Nginx Proxy Manager](https://nginxproxymanager.com)。

好的

**Reply by threerog** [reply]
> 也可以使用 [Nginx Proxy Manager](https://nginxproxymanager.com)。

我现在就是使用的[Nginx Proxy Manager](https://nginxproxymanager.com/)
但是仍然无法访问

具体怎么配置
